config/locales/en.yml

```diff
@@ -90,6 +85,12 @@ en:
       policies: "Policies"
       statistical_data_sets: "Statistical data sets"
       topical_events: "Topical events"
+    show_password:
+      show: Show
+      hide: Hide
```

Comment:
The way I probably would have done this is something like
```
show_visible_text: Show
show_hidden_text: Show password
.....
```
...and adjusted `show-password.js` to work with that.  I think I'd prefer that option for a few reasons:
- in case we have to translate this stuff into a language that does not have the same sentence structure as English 
- to support translations where a different form of a verb would be used when the verb is alone ("show") vs when it is accompanied by an object ("show password")
- to make this easier for devs to understand – I don't think it would immediately be clear why things are structured like this in this locale file


Then again what you've done works as well, and I don't think we're translating the Account manager into other languages anytime soon are we?

Reply:
I documented these possible problems already in the description. Unfortunately the words 'show' and 'password' need to be separate because one of them gets wrapped in a visually hidden span. I could add JS to split the words and insert them, but that would again be making assumptions about the language that would be hard to undo.

Reply:
I think that this could be solved by using `aria-label` to have 'show password' and just have 'show' in the element:

```
<button aria-label="Show password">Show</button>
```

This would also help for languages - such as the Spanish example - that appear to have differing words for 'show' depending on how they're used as they could be different keys in the localisation file.

Reply:
There we go - I knew there must be a better solution to this! Thanks @injms, have implemented. Will update the PR description to fit.